packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -447,7 +436,7 @@ const EAGivingPortalPage = ({classes}: {classes: ClassesType}) => {
                 <ul>
                   <li><b>Pre-vote</b> to show which candidates you're likely to vote for. Pre-votes are anonymous, don't turn into real votes, and you can change them at any time.</li>
                   <li><b>Add candidates</b> if you think they should be in the Election. Any project <a href="https://www.givingwhatwecan.org/">
-                  here </a> can be a candidate.</li>
+                  here</a> can be a candidate.</li>
```

Comment:
I think the link above should go here: https://www.givingwhatwecan.org/donate/organizations (bug from previous PR)

Reply:
Actually Lizka says it should go here: https://docs.google.com/spreadsheets/d/1I-IFdkai9frIIMO6fVqOIp6PDllXG713UhnI1WuwyiQ/edit?usp=sharing

Reply:
Got it, thanks!